app/screens/home/search/search.tsx

```diff
@@ -190,7 +190,7 @@ const SearchScreen = ({teamId, teams}: Props) => {
 
         setFileInfos(files?.length ? files : emptyFileResults);
         if (postResults.order) {
-            const postModels = await getPosts(serverUrl, postResults.order);
```

Comment:
Probably the result is going to be exactly the same, but I wonder if it should respect `postResults.order` instead of `create_at`.

Reply:
@larkox  the problem with that is that SQLite does not seem to respect the order of the ids in the query using `IN` if we want that, I think we need to do some sort of manual ordering using `WHEN` see https://stackoverflow.com/questions/56258201/sqlite-pull-items-in-the-same-order-as-in-array

Reply:
Or reorder them on JS after we get them from the database. But probably the result will be the same.

Reply:
yes, in that case, the result will be the same, but using JS we are using a single threaded environment to accomplish the same result while the DB will sort using as many threads as needed improving the performance.